src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -20,35 +20,36 @@
 public class LogsCenter {
     private static final int MAX_FILE_COUNT = 5;
     private static final int MAX_FILE_SIZE_IN_BYTES = (int) (Math.pow(2, 20) * 5); // 5MB
+    private static final String BASE_LOGGER_NAME = "ab3";
```

Comment:
I wonder if this can be omitted. As no one needs to know this detail except the baseLogger, it feels like we are 'over-exposing' it here.
Instead, we can specify it at the point we create the base logger, and access it from the baseLogger object in other places it is needed.
Is that possible/better?

Reply:
Ill try with this approach as we can retrieve the logger name via `logger::getName` but this may bring out things like "magic strings" within the function.